switchapi/src/switch_hostif.c

```diff
@@ -739,9 +743,10 @@ switch_api_hostif_rx_packet_from_hw(switch_packet_header_t *packet_header, char
 
     JLG(temp, switch_hostif_rcode_array, cpu_header->reason_code);
     if (!temp) {
-        SWITCH_API_ERROR("rx_packet w/ un-handled reason_code 0x%x\n",
-                            cpu_header->reason_code);
-        return SWITCH_STATUS_ITEM_NOT_FOUND;
+        JLG(temp, switch_hostif_rcode_array, SWITCH_HOSTIF_REASON_CODE_NONE);
+       // SWITCH_API_ERROR("rx_packet w/ un-handled reason_code 0x%x\n",
+       //                     cpu_header->reason_code);
+       // return SWITCH_STATUS_ITEM_NOT_FOUND;
```

Comment:
Ideally any non zero reason code has to be registered in order to receive the packet. For now, remove the changes here.